### PR TITLE
Make Layout container validation fail on non existent fields

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4694

## Description
Fixed an issue where **validate** passed a layouts container with incident fields not present in `id_set.json` even when `--post-commit` was indicated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 * Added **generate-unit-tests** documentation to the repo README.
 * Added the `hiddenpassword` field to the integration schema, allowing **validate** to run on integrations with username-only inputs.
 * Fixed an issue where **doc-review** required dot suffixes in release notes describing new content.
+* Fixed an issue where **validate** passed a layouts container with incident fields not present in `id_set.json` even when `--post-commit` was indicated.
```
Comment (YuvHayun) [inline CHANGELOG.md:12]:
```suggestion
* Fixed an issue where **validate** didn't recognize layouts with incident fields missing from `id_set.json` even when `--post-commit` was indicated.
```

**Review by YuvHayun — APPROVED** [review]
Nice, just a small comment.

**Review by idovandijk — COMMENTED** [review]
Good validation! Please check on some existing layouts and verify this, because in the past we had this https://jira-hq.paloaltonetworks.local/browse/CIAC-2445

Need to be sure that we don't start getting a lot of false positives

**Comment by DinaMeylakh** [comment]
@idovandijk I just fixed a bug where the validation always passed by mistake, the issue you attached was handled & the logic from the pr that fixes it still holds
https://github.com/demisto/demisto-sdk/pull/2099

Will check on existing layouts.

**Comment by idovandijk** [comment]
> @idovandijk I just fixed a bug where the validation always passed by mistake, the issue you attached was handled & the logic from the pr that fixes it still holds #2099
> 
> Will check on existing layouts.

Sounds good. Thanks for letting me know.

**Comment by DinaMeylakh** [comment]
Screen Shot 2023-04-23 at 18 53 54
Changed abusetype field id to abusetypeNOTTRUE and ran with master demsito-sdk version, which did not detect this. Ran with my version and it did :)

**Pull request merged**